bin/cheribsdtest/cheribsdtest.c

```diff
@@ -2440,20 +2450,23 @@ main(int argc, char *argv[])
 	sl_free(cheri_failed_tests, true);
 	sl_free(cheri_xfailed_tests, true);
 	sl_free(cheri_xpassed_tests, true);
-	if (tests_passed + tests_failed > 1) {
-		if (expected_failures == 0 && tests_xfailed == 0)
-			xo_emit("SUMMARY: passed {d:/%d} failed {d:/%d}\n",
-			    tests_passed, tests_failed);
-		else if (expected_failures == tests_xfailed)
-			xo_emit("SUMMARY: passed {d:/%d} failed {d:/%d} "
-			    "({d:/%d} expected {Np:failure,failures})\n",
-			    tests_passed, tests_failed, expected_failures);
-		else
-			xo_emit("SUMMARY: passed {d:/%d} failed {d:/%d} "
-			    "({d:/%d} expected {Np:failure,failures}) "
-			    "({d:/%d} unexpected {Np:pass,passes})\n",
-			    tests_passed, tests_failed, tests_xfailed,
-			    expected_failures - tests_xfailed);
+	if (tests_run > 1) {
+		xo_emit("{Lc:SUMMARY}");
+		sep = " ";
+#define	EMIT_SUMMARY_FIELD(label, value)				\
```

Comment:
Personally not keen on the weird tab after defined and apparently it's not required by style(8).

Reply:
"Put a single tab character between the #define and the macro name."?

Plus it keeps it aligned with the `#undef` later.

Reply:
I guess this could also be a function that takes label+sep+value instead of a macro (and then you wouldn't need to add another local variable), but the macro also seems fine.

Reply:
You need the local variable in case no tests passed.

Reply:
> "Put a single tab character between the #define and the macro name."?
> 
> Plus it keeps it aligned with the `#undef` later.
You're right, I could have sworn it wasn't specified.